software/winrm/src/main/java/org/apache/brooklyn/core/sensor/windows/WinRmCommandSensor.java

```diff
@@ -144,15 +144,7 @@ public T apply(String input) {
     public static String makeCommandExecutingInDirectory(String command, String executionDir, Entity entity) {
         String finalCommand = command;
         String execDir = executionDir;
-        if (Strings.isBlank(execDir)) {
-            // default to run dir
-            execDir = entity.getAttribute(BrooklynConfigKeys.RUN_DIR);
-            // if no run dir, default to home
-            if (Strings.isBlank(execDir)) {
-                execDir = "%USERPROFILE%";
-            }
-        }
-        if (!"~".equals(execDir)) {
+        if (Strings.isNonBlank(execDir) && !"~".equals(execDir)) {
```

Comment:
Default directory is `%USERPROFILE%`.
You can try by setting a sensor from `command: dir`.
We already use the same default directory for launch.command and other commands in `VanillaWindowsProcess`.